docs/source/guides/gpu-inference.rst

```diff
@@ -98,6 +79,25 @@ This means when building your Bento, you DO NOT need to specify ``cuda_version``
         - torch
         - tensorflow[and-cuda]
 
+If you know what you are doing and want to customize the installation of CUDA driver and libraries via ``system_packages``, ``setup_script``, or ``base_image`` options under the :ref:`docker-configuration` field.
+
+
+(Deprecated) If your Bento is using GPUs during deployment, configure its required CUDA version in the ``docker`` field of ``bentofile.yaml``. BentoML will install the corresponding version in the Docker image created. An example:
+
+.. code-block:: yaml
+
+    service: "service:GPUSVC"
+    labels:
+      owner: bentoml-team
+      stage: demo
+    include:
+    - "*.py"
+    python:
+      requirements_txt: "./requirements.txt"
+    docker:
+      python_version: "3.11.7"
```

Comment:
No config here is about GPU specifically, right? My understanding is that we don't want to use `cuda_version`. Maybe we can just remove these lines and let people [see the doc](https://docs.bentoml.com/en/latest/guides/build-options.html#docker-options-table) here.

Reply:
sure

Reply:
```suggestion
```